src/test/java/org/cactoos/map/MapOfTest.java

```diff
@@ -108,4 +108,15 @@ public void createsMapWithFunctions() {
         );
     }
 
+    @Test
+    public void string() {
```

Comment:
@SharplEr Can you rename this one: "integersToString"?

Reply:
Done